platforms/dotnet/versionsToBuild.txt

```diff
@@ -1,54 +1,9 @@
 # version, sha, download url
-1.1.14, E36AA4B278B936E1E2989B5E556F4EC3AEF2CB1EBB5ABD1BD6011D511873FC3E72101D035CF4DAF1563C7A3C43C193863D38CBF24288D375DD143AB42AFAE524, https://dotnetcli.blob.core.windows.net/dotnet/Sdk/1.1.14/dotnet-dev-debian.9-x64.1.1.14.tar.gz
 
-2.1.801, 5781C6E254270803D9BB2C29B1555FB9DB258BF99BA2EFA2B457CA28226C70D6305C9B7C141CBF49ADE4915E136FAE941727ADFC60373E4822C52BF0E92C451B,
```

Comment:
wondering why delete these versions?

Reply:
as discussed we wont be needing them in future .. so do not want to build them for buster and ubuntu versions .. we already have them in stretch